integration_tests/dbt_project.yml

```diff
@@ -1,9 +1,13 @@
 name: 'marketo_integration_tests'
-version: '0.11.0'
+version: '0.12.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
+    # enable when regenerating docs
+    marketo__enable_campaigns: true
+    marketo__enable_programs: true
+
```

Comment:
I just noticed the models config is not included in here. Can we add that for easier future validation testing.

```yml
models:
    +schema: "marketo_{{ var('directed_schema','dev') }}"
```

Reply:
Added.